.github/workflows/stale.yml

```diff
@@ -0,0 +1,23 @@
+name: "Stale label"
+on:
+  schedule:
+    - cron: '0 0 * * *'
+  branches:
+    - dev-es    
+jobs:
+  stale:
+    timeout-minutes: 1
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/stale@v7
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        stale-issue-message: 'This issue is stale because it has been labeled with inactivity.'
+        stale-pr-message: 'This PR is stale because it has been labeled with inactivity.'
```

Comment:
https://github.com/actions/stale#stale-issue-message

stale-issue-message
- Required Permission: issues: write

stale-pr-message
- Required Permission: pull-requests: write

It seems these options of this workflow requires write permissions.